Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -250,6 +251,12 @@ class LoginListViewController: SensitiveViewController {
         let deferred = Deferred<Maybe<[LoginRecord]>>()
         profile.logins.searchLoginsWithQuery(query) >>== { logins in
             deferred.fillIfUnfilled(Maybe(success: logins.asArray()))
+
+            // consider taking logins to breachalertsmanager from here
+            // which indexes need a special layout? to display breached login
+            breachAlertsManager.loadBreaches()
+            breachAlertsManager.compareToBreaches(logins.asArray())
+
```

Comment:
Maybe remove this portion for now, as I think handling the breach alerts result is a separate chunk of work for it's own PR and it's not fully finished here.